handbook/business-operations/README.md

```diff
@@ -486,41 +486,44 @@ Here are the steps hiring managers follow to get an offer out to a candidate:
+4. **Draft email:** 🦿 Apprentice to the CEO drafts the offer email in the CEO's inbox, reviews one more time, and then brings it to their next daily meeting for CEO's approval:
    - To: The candidate's personal email address _(use the email from the CEO interview calendar event)_
-   - Cc: Zach Wasserman and Head of Business Operations (both included as a recipient but will not participate in the email thread until after the offer is accepted)
+   - Cc: Zach Wasserman and Head of Business Operations _(neither participate in the email thread until after the offer is accepted)_
    - Subject: "Full time?"
-   - Body: _(The offer email is copied verbatim from Google Drive into Gmail as the body of the message, formatting and all.)_
+   - Body: _(The offer email is copied verbatim from Google doc into Gmail as the body of the message, formatting and all.  Check all links in offer letter for accuracy, and also click the surrounding areas to ensure no "ghost links" are left from previous edits... which has happened before.  Re-read the offer email one last time, and especially double-check that salary, number of shares, and start date match the equity plan.)_
+5. **Send offer:** 🐈‍⬛ CEO reviews and sends the offer to the candidate:
+   - _Grant the candidate "edit" access_ to their "exit scenarios" spreadsheet.
+   - _Send_ the email.
 
 ##### Steps after an offer is accepted
 
-Once the new team member replies and accepts their offer in writing, Head of Business Operations replies to the candidate.  The Head of Business Operations is then responsible for these steps:
+Once the new team member replies and accepts their offer in writing, 🔦 Head of Business Operations follows these steps:
 1. **Verify, track, and reply:** Reply to the candidate:
    - _Verify the candidate replied with their physical address… or else keep asking._  If they did not reply with their physical address, then we are not done.  No offer is "accepted" until we've received a physical address.
    - _Review and update the team database_ to be sure everything is accurate, **one last time**.  Remember to read the column headers and precisely follow the instructions about how to format the data:
      - The new team member's role in ["🧑‍🚀 Fleeties"](https://docs.google.com/spreadsheets/d/1OSLn-ZCbGSjPusHPiR5dwQhheH1K8-xqyZdsOe9y7qc/edit#gid=0) now includes:
-       - **Start date** _(the new fleetie's first day, YYYY-MM-DD)_
-       - **Location** _(derive this from the physical address)_
-       - **GitHub username**  _(username of 2FA-enabled GitHub account)_
+       - **Start date** _(The new fleetie's first day, YYYY-MM-DD)_
+       - **Location** _(Derive this from the physical address)_
+       - **GitHub username**  _(Username of 2FA-enabled GitHub account)_
+       - **@fleetdm.com email** _(Set this to whatever email you think this person should have)_
```

Comment:
@JoStableford connecting the dots
<img width="596" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/0327ae13-374e-4ff9-ad12-f7e14e77c389">